docs/examples.md

```diff
@@ -43,6 +88,17 @@ template using the following command line:
 dotnet new gcf-event
 ```
 
+This function expects Storage events, and logs details of the event it receives.
+
+Sample deployment:
+
+```sh
+gcloud functions deploy simple-event --runtime dotnet3 \
+  --trigger-event google.storage.object.finalize \
+  --trigger-resource <gcs-bucket-name> \
```

Comment:
I like to use `$GCS_BUCKET_NAME` such that I can just copy/paste shell commands.

On the docs we use _`YOUR_`_ like _`YOUR_TRIGGER_BUCKET_NAME`_

https://cloud.google.com/functions/docs/tutorials/storage#object_metadata_update_triggering_the_function

Either of those versions would be slightly easier to use.

Reply:
We can't use italics in Markdown code blocks. I've used GCS_BUCKET_NAME, but I've also added an explanatory comment at the start to indicate that you don't *need* those environment variables. I dislike guides that tell you to set environment variables but don't explain that it's just for the sake of the command lines shown - it leaves readers potentially confused about whether the tools themselves will use the environment variables (as they do for GOOGLE_APPLICATION_CREDENTIALS for example).